__tests__/app.test.js

```diff
@@ -102,3 +102,43 @@ describe("GET: /api/reviews", () => {
       });
   });
 });
+
+describe("GET: /api/reviews/:review_id/comments", () => {
+  it("404: when accessing a non-existant review", () => {
+    return request(app).get("/api/reviews/20/comments").expect(404);
+  });
+
+  it("404: when accessing a review with no comments", () => {
```

Comment:
Have a think about whether or not this circumstance should be an error, the review exists and therefore the user input was valid, but it just so happens that review has no comments. Do we want to say that's an error? Or just say here you go - there's no comments for that review.